aws/main.tf

```diff
@@ -279,13 +279,43 @@ chmod 600 /home/${var.default_ssh_user}/.ssh/authorized_keys
 
 # Install NFS utilities and mount the EFS file system at /opt/thirdai_platform/model_bazaar
 
+sleep 60
+
 # Determine package manager and install necessary NFS utilities
 if [ "${var.default_ssh_user}" == "ubuntu" ]; then
-  apt install -y nfs-common
+  apt update
+  apt install -y nfs-common postgresql-client
 else
-  yum install -y amazon-efs-utils nfs-utils
+  amazon-linux-extras enable postgresql14
+  yum install -y amazon-efs-utils nfs-utils postgresql postgresql-server
 fi
 
+echo "Testing DNS resolution..." >> /var/log/user_data.log
```

Comment:
Yeah true it would be by default in output.log place, but that log is easily gets crowded due to that download and platform installation, whenever i want to see something specific it was very hard to find, so i just added this so that in future if we want to log any particular things and see that output without worrying about where the logs, this way we can easily see, i dont have strong opinion on either of ways, its just that this way it feels convenient.